openmetadata-spec/src/main/resources/json/schema/entity/docStore/document.json

```diff
@@ -10,7 +10,12 @@
     "data": {
       "javaType": "org.openmetadata.schema.entities.docStore.Data",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": true,
```

Comment:
here if we are making data is of type emailTemplate, is that right?

Reply:
data is a generic data object that can store any type of data as per our need

Reply:
>> here if we are making data is of type emailTemplate, is that right? 
no, I've created a new key emailTemplate inside data, and set its type as  emailTemplate.

Reply:
why data would have nameKey emailTemplate, it can store anything , we have created our type emailTemplate.json, but we cannot hardbind that on data saying data will have a name key emailTemplate, does all data exhibit this property .. no !

Reply:
You are correct. I hadn't considered that. I am updating the code accordingly.